data/manual.json

```diff
@@ -270,6 +270,14 @@
         "motion",
         "editing-support"
       ]
+    },
+    {
+      "type": "github",
+      "username": "Bekaboo",
```

Comment:
missing "repo" key

Reply:
Sorry for my ignorance, it should be fixed now.